packages/tea-map-cache/index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-console */
+const AWS = require('aws-sdk');
```

Comment:
This should be `import AWS from 'aws-sdk';`

Reply:
I don't see a direct reason in this case we can't use a ES6 style import, but is there a technical reason we *should* in a node file?    I ask, as every other use of `aws-sdk` in our project via node is using Common.

Reply:
Sorry, you're correct. I had the other fix in my head.